public/css/leaderboard-table.css

```diff
@@ -29,3 +29,9 @@
 .col-isuser {
   font-weight: bold;
 }
+
+img.responsive-img, video.responsive-video {
```

Comment:
what does this do? also this css and the next css file have different indentation, please fix

Reply:
This is css of the pictures in table besides the students name. I used in the responsive class but had to tweek it.